app/editor/src/features/content/list-view/constants/columns.tsx

```diff
@@ -46,10 +46,10 @@ export const columns: (Column<IContentModel> & UseSortByColumnOptions<IContentMo
     accessor: (row) => row.status,
   },
   {
-    id: 'publishedOn',
+    id: 'createdOn',
```

Comment:
@jdtoombs I just changed this because the `createdOn` doesn't seem to be useful unless the `publishedOn` is null.  Is there a reason you overwrote?

Reply:
Ugh no my rebase was all messed one second